### ISSUE TITLE
Unable to see build log sometimes

### ISSUE THREAD

**Issue by jakirkham** [travis-ci/travis-ci#5334]
Using Mac OS 10.9.x and Firefox 43.x, I see the following.

<img width="1554" alt="screen shot 2015-12-18 at 16 31 56" src="https://cloud.githubusercontent.com/assets/3019665/11907813/1c6016e0-a5a5-11e5-9391-1d619d447973.png">

**Comment by jakirkham** [comment]
@BanzaiMan, here is the issue with the log not appearing.

**Comment by BanzaiMan** [comment]
https://travis-ci.org/nanshe-org/nanshe/builds/97752458 (the raw log is https://s3.amazonaws.com/archive.travis-ci.org/jobs/97752459/log.txt) either has logs out of order, or contains some ANSI control sequence(s) that is messing up the logs. If you click on the left arrow, you can see the logs, but it is not clear to me how we got into this situation.

I suspect rerunning would fix the display problem.

**Comment by jakirkham** [comment]
This log seems to have the same issue now ( https://travis-ci.org/ukoethe/vigra/jobs/105184100 ).

**Comment by joepvd** [comment]
It looks like that in the linked build logs, the job ran twice in parallel. There recently was a fix that addresses a possible cause, we assume that this should not happen again. Please open a new ticket when you would encounter a similar situation.

**Comment by jakirkham** [comment]
How recent are we talking? I have two [builds]( https://travis-ci.org/conda-forge/conda-forge.github.io/builds/166551357 ) just like this from 2 days ago. The screenshot of one of them is shown below.

<img width="1554" alt="screen shot 2016-10-14 at 1 24 42" src="https://cloud.githubusercontent.com/assets/3019665/19376601/0a1f8f7a-91ad-11e6-8245-9901ec75dbcc.png">

**Comment by joepvd** [comment]
Thanks. This seems to be another cause. Reopening and investigating the recent case.

**Comment by svenfuchs** [comment]
@jakirkham Sorry about this. We've had a bug that caused a lock to not work as expected, and that caused your build to be scheduled twice. I have just deployed a fix for this. 

I am going to close this one here. Please reopen if you still see the issue.

**Comment by jakirkham** [comment]
Alright, then what about these [3 logs]( https://travis-ci.org/conda-forge/boost-feedstock/builds/168178211 ) all with the same issue?

**Comment by joepvd** [comment]
Thanks! You have been running into another issue. We are looking into this.

**Comment by jakirkham** [comment]
FWIW I appreciate all the effort you are putting into this issue. It's definitely nice to hear that we are eliminating potential causes.

**Comment by backspace** [comment]
We have a fix for the mis-displaying log that you can try out [at this deployment](https://bd-double-fold.test-deployments.travis-ci.org/), but as you can see there, the doubled job is a separate problem. I expect to be able to deploy the log fix next week.

**Comment by backspace** [comment]
I’ve deployed that fix. Do you want to close this, @jakirkham, or change its name to be about the multiple workers instead? 😆
